samples/net/wpan_serial/README.rst

```diff
@@ -0,0 +1,54 @@
+.. _wpanusb_serial-sample:
+
+802.15.4 "serial-radio" sample
+##############################
+
+Overview
+********
+
+The wpan_serial sample shows how to use hardware with 802.15.4 radio and USB
+controller as a "serial-radio" device for Contiki - based border routers.
+
+Requirements
+************
+
+The sample assumes that 802.15.4 radio and USB controller are supported on
+a board.
+
+Building and Running
+********************
+
+#. Build and flash the sample Zephyr application to a board
+   with a 802.15.4 radio and USB controller
+   (such as quark_se_c1000_devboard)
+#. Connect board to Linux PC, /dev/ttyACM[number] should appear.
+#. Run Contiki-based native border router (6lbr, native-router, etc)
+   Example for Contiki:
+
+   .. code-block:: console
+
+     $ cd examples/ipv6/native-border-router
+     $ make
+     $ sudo ./border-router.native -v5 -s ttyACM0 fd01::1/64
+
+Now you have Contiki native border router, simple web server should show some
+802.15.4 devices. Web server address is printed in the border-router output
+screen:
+
+.. code-block:: console
+
+  ...
+  Server IPv6 addresses:
+   0x62c5c0: =>fd01::212:4b00:531f:113a
+  ...
+
+Go to http://[fd01::212:4b00:531f:113a]/
+
+It shall show you Neighbors and Routes.
+
+PS: Make sure your disto ModemManager is disabled, otherwise it sends AT commands
```

Comment:
This "step" should be earlier in the instructions.  Delete this here are add the step noted above.